sdk/src/AbstractMain.cpp

```diff
@@ -220,6 +219,13 @@ int AbstractMain::run(int in_argc, char** in_argv)
    if (ret != 0)
       return ret;
 
+   // If we are, restore root privileges.
+   if (!options.useUnprivilegedMode() && system::posix::realUserIsRoot())
+   {
+   error = system::posix::restoreRoot();
```

Comment:
nit: need more whitespace here

Reply:
> Just a few whitespace nits, but otherwise I think this is ready!

I think the last one may be an older commit. It is not showing on the branch. And awesome, I've reached out to Maria.